packages/components/src/components/checkbox/checkbox.scss

```diff
@@ -1,8 +1,10 @@
 .dc-checkbox {
+    align-items: center;
     display: flex;
-    justify-content: flex-start;
     cursor: pointer;
-    align-items: center;
+    justify-content: flex-start;
```

Comment:
@artsemkavaleu-deriv  Please don't change the components package code, that's reusable code.

Reply:
sure, thx for notice!